webrender/src/device/gfx/render_pass.rs

```diff
@@ -35,4 +36,134 @@ impl<B: hal::Backend> RenderPass<B> {
             device.destroy_render_pass(self.bgra8_depth);
         }
     }
+
+    pub fn create_render_passes(
+        device: &B::Device,
+        surface_format: hal::format::Format,
+        depth_format: hal::format::Format,
+    ) -> HalRenderPasses<B> {
+        let attachment_r8 = hal::pass::Attachment {
+            format: Some(hal::format::Format::R8Unorm),
+            samples: 1,
+            ops: hal::pass::AttachmentOps::new(
+                hal::pass::AttachmentLoadOp::DontCare,
```

Comment:
Note: `DontCare` is not entirely wrong for our end game (we should either `Clear` it or not care), but beware that it is incompatible with the scheme you are temporarily using: `end_render_pass()` followed by `begin_render_pass()`, since the latter will technically invalidate the previous contents.

Reply:
Changed it to `Load` for now, I will revisit these in the next steps.